doc/ansible/dev/advanced_options.md

```diff
@@ -50,3 +50,45 @@ You may want to manage what your operator watches and the owner references. This
 ```
 ENTRYPOINT ["/usr/local/bin/entrypoint", "--inject-owner-ref=false"]
 ```
+
+## Max Workers
+
+Increasing the number of workers allows events to be processed
+concurrently, which can improve reconciliation performance.
+
+Worker maximums can be set in two ways. Operator **authors and admins**
+can set the max workers default by including extra args to the operator
+container in `operator.yaml`. (Otherwise, the default is 1 worker.)
+
+**NOTE:** Admins using OLM should use the environment variable instead
+of the extra args.
+
+``` yaml
+- name: operator
+  image: "quay.io/asmacdo/memcached-operator:v0.0.0"
+  imagePullPolicy: "Always"
+  args:
+    - "--max-workers"
+    - "3"
+```
+
+Operator **admins** can override the value by setting an environment
+variable in the format `WORKER_<kind>_<group>`. This variable must be
```

Comment:
Since there is some mutation of the kind/group might be worth it to explain that, ie replace dots with underscores, all caps. Perhaps show the unmutated group and kind for your example resource, and then the result of the mutation, and then go on to show the example snippet below.